docker-compose.yaml

```diff
@@ -190,6 +192,8 @@ services:
     image: thecodingmachine/nodejs:16
     command: yarn start:dev
     #command: yarn run profile
+    ports:
+      - 3000:3000
```

Comment:
Do we need to expose port 3000 on the host?
The Traefik label should be enough to access the environment only on map-storage.workadventure.localhost

Reply:
Without exposing I get `ERR_CONNECTION_REFUSED` when trying to get the map file, for reasons :/